src/renderer/features/operation/sign/ui/WalletConnect/WalletConnect.tsx

```diff
@@ -99,28 +106,20 @@ export const WalletConnect = ({
   const signTransaction = async () => {
     if (!api || !client || !session) return;
 
-    try {
-      const payload = {
-        // eslint-disable-next-line i18next/no-literal-string
-        chainId: `polkadot:${transaction.chainId.slice(2, 34)}`,
-        topic: session.topic,
-        request: {
-          method: DEFAULT_POLKADOT_METHODS.POLKADOT_SIGN_TRANSACTION,
-          params: {
-            address: transaction.address,
-            transactionPayload: unsignedTx,
-          },
+    const payload = {
```

Comment:
Seems to me, this way we'll have TS support because of `createEvent<SignParams>();` otherwise we can make a mistake
```javascript
walletConnectSignModel.events.signingStarted({
  client,
  payload: { ... },
});
```